app/models/division.rb

```diff
@@ -21,7 +21,30 @@ class Division < ActiveRecord::Base
   has_closure_tree
   alias_attribute :super_division, :parent
 
-  belongs_to :organization
+
+  #JE: I like to keep a reference for the implicit db attributes here in the model class
+  # create_table :divisions do |t|
+  #   t.references :organization, index: true, foreign_key: true
+  #   t.string :name
+  #   t.text :description
+  #   t.integer :parent_id
+  #   t.references :currency, index: true, foreign_key: true
+  #   t.timestamps null: false
+  # end
```

Comment:
Just make sure to yank the schema comment here since we're automating those now.